src/llmq/quorums_blockprocessor.cpp

```diff
@@ -321,7 +321,8 @@ bool CQuorumBlockProcessor::GetCommitmentsFromBlock(const CBlock& block, const C
                 return state.DoS(100, false, REJECT_INVALID, "bad-qc-dup");
             }
 
-            ret.emplace((Consensus::LLMQType)qc.commitment.llmqType, std::move(qc.commitment));
+            auto temp = (Consensus::LLMQType) qc.commitment.llmqType;
```

Comment:
don't name variable `temp` please. `llmqType` is much better name

Reply:
Use of the name temp is an indicator that the variable is semantically required. Can document in a comment instead if that's preferable.